puppetwash.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |spec|
   spec.description   = "A Wash plugin for examining Puppet infrastructure nodes such as PuppetDB, Puppet Server, and PE Classifier"
   spec.homepage      = "https://github.com/puppetlabs/puppetwash"
   spec.license       = "Apache-2.0"
-  spec.files         = Dir["*.rb"] + ["puppet"]
+  spec.files         = Dir["*.rb"]
   spec.require_paths = ["."]
```

Comment:
Boltwash doesn't have this. Do we still need it here?

Reply:
We should remove it. `require_paths` would mean that anything in this gem could be found when you `require`, as in `require 'puppet'` might pick it up instead of the puppet gem.

Reply:
The absence is also why I use require_relative to find other files in bolt.rb.